test/configCases/library/function-umd/webpack.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+	output: {
+		libraryTarget: () => "umd",
+		library: () => ({
```

Comment:
The test case should feature multiple entrypoints which have different library values per name.

i. e. `library: ({ chunk }) => chunk.name`